### ISSUE TITLE
Rename `export_file` to `export_table_to_file`

### ISSUE THREAD

**Issue by kaxil** [astronomer/astro-sdk#906]
Context: https://docs.google.com/document/d/1OdS1o2YoL3u1VNO8G9qPi-cb1dz5zn50rPHvmSTikBo/edit?usp=sharing

**Comment by kaxil** [comment]
Deprecate please

**Comment by rajaths010494** [comment]
duplicate of https://github.com/astronomer/astro-sdk/issues/1530

**Comment by sunank200** [comment]
This can be closed

**Comment by pankajastro** [comment]
No, this can't be closed. We just deprecated the `export_file` and we will remove it in 2.0.0
